### PR TITLE
Update GitHub workflows

### PR DESCRIPTION
**Pull request by annda**
The new workflow also checks code style, so some fixes are included in this PR.

### CODE REVIEW

File: Functions.php [hunk 1]
```diff
@@ -43,6 +42,7 @@ public static function register($pdo)
      * @param string $string column value
      * @param string $separator separator added between values
      */
+    // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
```
Comment (splitbrain) [inline Functions.php:45]: any reason why we can't rename this method to `groupConcatStep`?

File: Functions.php [hunk 2]
```diff
@@ -65,6 +65,7 @@ public static function GroupConcatStep(&$context, $rownumber, $string, $separato
      * @param int $rownumber number of rows over which the aggregate was performed.
      * @return null|string
      */
+    // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
```
Comment (splitbrain) [inline Functions.php:68]: same here?

**Review by splitbrain — APPROVED** [review]

File: Functions.php (hunk 1 above)
Comment (annda) [reply Functions.php]: No good reasons, except that I generally avoid renaming public methods. But those two are safe, so I've just renamed them.

File: Functions.php (hunk 2 above)
Comment (annda) [reply Functions.php]: done

**Pull request merged**